webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -392,7 +392,7 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
                         />
                     </a>
                 </div>
-                <div className='get-app__preference'>
+                <label className='get-app__preference'>
```

Comment:
It is not a recommended practice to have button inside of a label. https://html.spec.whatwg.org/multipage/forms.html#the-label-element

if you would like you can improve all of this by good measure

We can instead use with correct styling! And we might have to get rid of the custom icon in favor of html checkbox

```
label
-- checkbox
-- text
label
```

Let me know if you would like to make improvements

Reply:
I definitely agree that it would be much cleaner to have a proper checkbox instead of a button masquerading as a checkbox. I have added commit 5667b35 which does that.